pages/documentation/guidance/accessibility.md

```diff
@@ -23,117 +25,146 @@ changelog:
+
+{:.border-top-1px.padding-top-2.measure-2}
+### We include accessibility-focused tools to build new functionality
+
+- Colors with [magic number contrast](https://designsystem.digital.gov/design-tokens/color/overview/#magic-number)
+- Rem-based size [tokens](https://designsystem.digital.gov/design-tokens/) and an em-based media query mixin that respond to user scale settings
+- Normalized [typography tokens](https://designsystem.digital.gov/design-tokens/typesetting/overview/) that provide reliable, readable results regardless of the typeface
+
+{:.border-top-1px.padding-top-2.measure-2}
+### We feature context-aware settings that allow accessible customization with minimal custom code
+
 - Adding more smart components that change their color based on background
 - Smart settings for links to automatically choose accessible link and hover states
 
-{:.border-top-1px.padding-top-2.margin-top-2.measure-2}
-### Make accessibility one of our core design principles
-- It's not only part of our mission to embrace accessibility in our own work, but to do what we can to encourage it.
-- **We prioritize accessibility issues.** We will work hard to fix accessibility-related issues quickly, and to get them into the next available release. Accessibility is a mission-critical service for USWDS, and we want to know when there's an opportunity to fix something that's broken or improve something that could be better.
+{:.border-top-1px.padding-top-2.measure-2}
+### We make accessibility one of our core design principles
+
+- We strive not only to embrace accessibility but also to inspire all federal web teams to embrace it, too.
+- **We prioritize accessibility issues.** We work hard to fix accessibility-related issues quickly, and to get them into the next available release. Our developers work with our accessibility specialists to review fixes with automated and manual testing.
 
-While we conduct extensive research during the design and development of components, it is impossible to predict every use case for accessibility and usability overall. That is why we encourage you to [contribute to the community]({{ site.baseurl }}/about/community) by providing feedback on how a component is working on your site. We also recommend performing your own user testing at the page level to gauge the usability of a component within context. We welcome any and all findings and feedback as we work to identify and prioritize improvements to our components and guidance.
+While we conduct research during the design and development of components, it is impossible to predict every use case for accessibility and usability. That is why we encourage you to contribute to the community by providing feedback on how a component is working on your site. We also expect you to perform your own user testing at the page level to gauge the usability of a component within context. We welcome any and all findings and feedback.
```

Comment:
## List with periods

This is the only list where each item ends in periods. Can we remove for consistency?

(GH won't let me link a suggestion due to removed lines)

Reply:
You mean the items under "We make accessibility one of our core design principles," right? We should leave the periods because this list is different than the earlier lists in that is made up only of complete sentences/groups of sentences.

Reply:
Sounds good to me! Marking as resolved 👍